tests/telemetry/telemetry_utils.py

```diff
@@ -77,6 +77,19 @@ def restore_telemetry_forpyclient(duthost, default_client_auth):
         duthost.service(name="telemetry", state="restarted")
 
 
+def check_gnmi_cli_running(ptfhost):
+    program_list = ptfhost.shell("ps aux | grep py_gnmicli.py")["stdout"]
+    matches = re.findall('python /root/gnxi/gnmi_cli_py/py_gnmicli.py', program_list)
```

Comment:
>re [](http://example.com/codeflow?start=14&length=2)

For plain text search, we do not need regex.

Reply:
Will change

Reply:
No need, passed pattern to pgrep, check stdout array size